torch_audiomentations/augmentations/shift.py

```diff
@@ -71,8 +71,10 @@ def randomize_parameters(
             <= max_shift_in_samples
             <= torch.iinfo(torch.int32).max
         )
-
-        selected_batch_size = selected_samples.size(0)
+        if selected_samples.dim() == 2:
+            selected_batch_size = 1
```

Comment:
If the tensor is two-dimensional, it it assumed that the audio inside it is mono and the first dimension is the batch size.
This means that we should do `selected_batch_size = selected_samples.size(0)` even when the tensor is two-dimensional

Reply:
Ah ok I was testing on multi channel 2d samples. I think it would be generally better to always expect a `b c t` shape. I think you said that somewhere too?

Reply:
Yeah, that does makes sense. It's very small price to pay for those who do only mono audio, not multichannel.

Reply:
What would you like me to change for this PR then?

Reply:
You can let `Shift` assume that the input tensor is always 3-dimensional